BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/Skonto/Views/SkontoAppliedHeaderView.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  SkontoAppliedHeaderView.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import UIKit
+
+class SkontoAppliedHeaderView: UIView {
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        let title = NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.info.title",
+                                                             comment: "With Skonto discount")
+        label.text = title
+        label.accessibilityValue = title
+        label.textColor = .giniColorScheme().text.primary.uiColor()
+        label.font = configuration.textStyleFonts[.bodyBold]
+        label.adjustsFontForContentSizeCategory = true
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+
+    private lazy var statusLabel: UILabel = {
+        let label = UILabel()
+        let title = NSLocalizedStringPreferredGiniBankFormat("ginibank.skonto.info.status",
+                                                             comment: "• Active")
+        let font = configuration.textStyleFonts[.footnoteBold] ?? .systemFont(ofSize: 17)
```

Comment:
[clarfification] Do we really need this `?? .systemFont(ofSize: 17)`? 
Is not confirmed to what we have:
` .footnoteBold: UIFontMetrics(forTextStyle: .footnote).scaledFont(for: UIFont.boldSystemFont(ofSize: 13))`